### PR TITLE
Add email about SR2021 state of affairs

### PR DESCRIPTION
**Pull request by RealOrangeOne**
This is an email simply to update teams on the fact we're currently planning what SR2021 might look like.

It also reiterates kits, just in case team leaders forgot or misplaced the original information (this also goes to all teams so may catch previous years)

### CODE REVIEW

File: SR2021/2020-09-25-state-of-affairs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+subject: Student Robotics Update
+---
+
+Hi All,
+
+Just a quick update on the current state of affairs with Student Robotics.
+
+## Student Robotics 2021
+
+Due to recent events, and the current measures in place, **We will not be running a physical competition for Student Robotics 2021**.
```
Comment (Adimote) [inline SR2021/2020-09-25-state-of-affairs.md:12]: just bold 'Physical', otherwise I think a skim read would make people think SR is cancelled

**Review by Adimote — APPROVED** [review]

File: SR2021/2020-09-25-state-of-affairs.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+subject: Student Robotics Update
+---
+
+Hi All,
+
+Just a quick update on the current state of affairs with Student Robotics.
+
+## Student Robotics 2021
+
+Due to recent events, and the current measures in place, We will not be running a **physical** competition for Student Robotics 2021.
```
Comment (sersorrel) [inline SR2021/2020-09-25-state-of-affairs.md:12]:
```suggestion
Due to recent events, and the current measures in place, we will not be running a **physical** competition for Student Robotics 2021.
```

Comment (rgilton) [inline SR2021/2020-09-25-state-of-affairs.md:12]: What does this mean?  Does it mean that there will be no robots, or does it mean that people will not be attending in person?

Comment (RealOrangeOne) [reply SR2021/2020-09-25-state-of-affairs.md:12]: Mostly the latter, although it's left intentionally vague to allow the former to be a possibility, should it be decided.

File: SR2021/2020-09-25-state-of-affairs.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Hi All,
+
+Just a quick update on the current state of affairs with Student Robotics.
+
+## Student Robotics 2021
+
+Due to recent events, and the current measures in place, we will not be running a **physical** competition for Student Robotics 2021.
+
+This does not mean there will be no competition! We are currently actively planning what Student Robotics 2021 will look like, and hope to have an update for you soon. Our current intention is that it would start around mid-late November.
```
Comment (raccube) [inline SR2021/2020-09-25-state-of-affairs.md:14]: >hope to have an update for you soon

This sounds a bit uncertain to me.

Comment (RealOrangeOne) [reply SR2021/2020-09-25-state-of-affairs.md:14]: It's unfortunately vague, but by design. Planning is ongoing, and exactly when we'll have an update for them is also unknown

Comment (raccube) [reply SR2021/2020-09-25-state-of-affairs.md:14]: Fair enough.

**Review by raccube — DISMISSED** [review]

**Pull request merged**